kubernetes/k8_create.yml.erb

```diff
@@ -0,0 +1,60 @@
+# Deployment for create and redis service
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: create-deployment
+  labels:
+    app: create
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: create
+  template:
+      metadata:
+        labels:
+          app: create
+      spec:
+        containers:
+          - name: create-worker
+            image: "docker.ocf.berkeley.edu/create:<%= version %>"
+            imagePullPolicy: Always
```

Comment:
What is the reasoning for this?

Reply:
Kubernetes does not always pull the latest image from the docker repo, so if we update the image and then redeploy the service, kube will keep the old image instead of re-pulling. Not sure how necessary this is for production, but it was painful for debugging.

Reply:
All of the images use a version which Jenkins will bump, so I don't think this is necessary.

Reply:
In production all the versions will be distinct so the same tag will never update